plugins/internal/telemetry/annotations.go

```diff
@@ -0,0 +1,31 @@
+package telemetry
+
+import (
+	"context"
+	"net/http"
+)
+
+var ctxKey = ctxKeyType("telemetry_annotations")
```

Comment:
What do you think of using `Labeler` from https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/net/http/otelhttp/labeler.go?

Reply:
done